.grit/patterns/react_to_hooks.md

```diff
@@ -50,7 +88,11 @@ pattern handle_one_statement($class_name, $statements, $states_statements, $stat
             and {
                 $statement <: prepend_comment($statements),
                 $async <: `async`,
-                $statements += `const ${name}Handler = useCallback(async $parameters => $body, []);`
+                if ($handler_callback_suffix <: .) {
+                  $statements += `const ${name} = async $parameters => $body;`
+                } else {
+                  $statements += `const ${name}${handler_callback_suffix} = useCallback(async $parameters => $body, []);`
```

Comment:
From my understanding one of the alternative cases we'd have to support is that these shouldn't become `useCallback`s at all but just bare functions, or are we not doing that?

Reply:
That's what setting `handler_callback_suffix` to null is fore.